spec/requests/api/v1/data_studio/school_rollout_spec.rb

```diff
@@ -80,6 +80,6 @@
   end
 
   def school_rollout_attributes
-    %i[urn name sent_at opened_at notify_status induction_tutor_nominated tutor_nominated_time induction_tutor_signed_in induction_programme_choice programme_chosen_time in_partnership partnership_time partnership_challenge_reason partnership_challenge_time lead_provider delivery_partner chosen_cip cip_chosen_time]
+    %i[urn name sent_at opened_at notify_status induction_tutor_nominated tutor_nominated_time induction_tutor_signed_in induction_programme_choice programme_chosen_time in_partnership partnership_time partnership_challenge_reason partnership_challenge_time lead_provider delivery_partner]
```

Comment:
Are we not interested in that? Or is it too expensive to run here?

Reply:
I believe those are no longer needed. The underlying attributes and the query to fetch them came from Chris.